pkg/openevec/eve.go

```diff
@@ -23,7 +23,7 @@ import (
 
 const SdnStartTimeout = 3 * time.Minute
 
-func StartEve(vmName string, cfg *EdenSetupArgs) error {
+func StartEve(vmName, tapInterface string, cfg *EdenSetupArgs) error {
```

Comment:
Not sure, should we do it in the PR, but probably it would be nice to keep `cfg *EdenSetupArgs` option in the beginning (I mean to have the same order at least for one option)?

Reply:
Or even better, we can define something like
```
func CreateOpenEVEC(cfg *EdenSetupArgs) *OpenEVEC
func (c *OpenEVEC) StartEve(vmName, tapInterface string) error
```